pkgs/development/python-modules/dask-yarn/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pythonOlder
+, distributed
+, dask
+, grpcio
+, skein
+}:
+
+buildPythonPackage rec {
+  pname = "dask-yarn";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "dask";
+    repo = pname;
+    rev = version;
+    hash = "sha256-/BTsxQSiVQrihrCa9DE7pueyg3aPAdjd/Dt4dpUwdtM=";
+  };
+
+  # Needed by tests
+  HOME = "/tmp";
```

```suggestion
  HOME = "$TMPDIR";
```
Also please move to preCheck with export.